### ISSUE TITLE
How do I understand the final output?

### ISSUE THREAD

**Issue by jmsong2** [caulai/Mo17_genome_assembly#2]
Hi,
    When I get the clustalw.out.checked.filtered.structure file, how can I do statistics for it? For example, the number of gene with large structural variations.
    Thanks a lot!

Regards,
Jiaming

**Comment by caulai** [comment]
Hi, I have upload a script '5.Gene_SV_classify.pl' to do statistics.
Thank you for running these scripts. If you meet problems when using running scripts, you can also email me. （yencechow@qq.com）

**Comment by jmsong2** [comment]
Thank you very much. I will try to it. And these scripts will report many "uninitialized value" warnings. I am not sure whether it will influence the result?

**Comment by caulai** [comment]
Nope, the warnings won't influence the result. But I suggest you use the updated scripts and it runs faster.

**Comment by jmsong2** [comment]
Well. I will re-run with updated scripts.

**Comment by jmsong2** [comment]
BTW, the 'gene.fa' (1.bwamem-main.pl) is expected to be a single sequence fasta splited from 'gene-full-cds-double.new.positive.fa' or the whole 'gene-full-cds-double.new.positive.fa' file.

**Comment by caulai** [comment]
You can use the whole  'gene-full-cds-double.new.positive.fa' file as ‘gene.fa’. But if your reference genome is too large, you can divide the 'gene-full-cds-double.new.positive.fa' file into ten or more small files and run separately  to make the work finished in several hours

**Comment by jmsong2** [comment]
Thanks for your advice!
